src/EditorFeatures/VisualBasic/LineCommit/CommitFormatter.vb

```diff
@@ -51,6 +51,9 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.LineCommit
                 End If
 
                 Dim textSpanToFormat = spanToFormat.Span.ToTextSpan()
+                If AbortForDiagnostics(document, textSpanToFormat, cancellationToken) Then
```

Comment:
I would assume we should not abort this if isExplicitFormat, which is set if somebody selected code and said to format. ("Doctor, it hurts if I do this!" "Don't do it.")